package.json

```diff
@@ -272,7 +272,7 @@
     "test-server": "jest -c=test/server/jest.config.js",
     "test-server:coverage": "npm run -s test-server -- --coverage",
     "test-server:watch": "npm run -s test-server -- --watch",
-    "translate": "i18n-calypso --format pot --output-file ./calypso-strings.pot -k translate,__,_x,_n,_nx -e date \"**/*.js\" \"**/*.jsx\" \"!build/**\" \"!public/**\" \"!node_modules/**\" \"!**/node_modules/**\" \"!packages/i18n-calypso/_test/**\" # FIXME: when tests are restored",
+    "translate": "babel --config-file ./i18n.babel.config.js client && find build/i18n-calypso -name \\*.pot | msgcat -f- -o calypso-strings.pot",
```

Comment:
Looks like we're running babel only on the `client/` folder here. Are we sure that we don't use `translate()` outside that folder?

Reply:
At this moment, only `client/` contains translated strings. In near future, when we start publishing UI components as NPM packages, we'll also need to include `packages/`.